dokka-subprojects/plugin-base/src/test/kotlin/translators/KMPTest.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package translators
+
+import org.jetbrains.dokka.model.*
+import utils.AbstractModelTest
+import java.io.File
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class KMPTest : AbstractModelTest("/src/main/kotlin/kmp/Test.kt", "kmp") {
+
+    // copy-pasted org.jetbrains.dokka.analysis.test.api.util.getResourceAbsolutePath
+    private fun getResourceAbsolutePath(resourcePath: String): String {
+        val resource = object {}.javaClass.classLoader.getResource(resourcePath)?.file
+            ?: throw IllegalArgumentException("Resource not found: $resourcePath")
+
+        return File(resource).absolutePath
+    }
+
+    val configuration = dokkaConfiguration {
+        sourceSets {
+            sourceSet {
+                sourceRoots = listOf("src/androidMain/kotlin")
+                analysisPlatform = "jvm"
+                name = "android-example"
+                classpath = listOf(getResourceAbsolutePath("jars/jvmAndroidLib-jvm-copy.jar"))
```

Comment:
would be nice to left some comment on how those jars were generated.

Reply:
I left a comment about  what jars contain. I think it is not a problem  to build such jars.